doc/source/conf.py

```diff
@@ -169,7 +169,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+# html_static_path = ['_static']
```

Comment:
Why has this been commented out?

Reply:
Because as far as I can see, we don't use it and the fact that it doesn't exist raises a warning.